src/visualization/components/internal/HoverLegend.tsx

```diff
@@ -179,8 +181,18 @@ const HoverLegend: FC<HoverLegendProps> = ({properties, update}) => {
   }
 
   const handleSetOrientation = (threshold: number): void => {
+    let validThreshold = threshold
```

Comment:
is this assignment necessary since it's an `if/else` ?

i think it should probably look closer to:

```ts
    let validThreshold
    if (
      typeof threshold !== 'number' ||
      threshold !== threshold ||
      threshold > 0
    ) {
      validThreshold = LEGEND_ORIENTATION_THRESHOLD_HORIZONTAL
    } else {
      validThreshold = LEGEND_ORIENTATION_THRESHOLD_VERTICAL
    }

Reply:
Yeah this reads better.